packages/commerce-sdk-react/src/constant.ts

```diff
@@ -13,3 +13,6 @@ export const IFRAME_HOST_ALLOW_LIST = Object.freeze([
     'https://runtime-admin-staging.mobify-storefront.com',
     'https://runtime-admin-preview.mobify-storefront.com'
 ])
+
+export const slasSecretWarningMsg =
```

Comment:
We are only ever showing this warning when we in fact are executing within the browser.. so we don't have to say "this function can run client-side".

`You are potentially exposing SLAS secret, make sure to keep it safe and secure!`

Reply:
Lets continue using the pattern of having our constants in all UPPERCASE_FORMAT so they can be readily distinguished from lower case variable names. 

```
SLAS_SECRET_WARNING_MSG
```